src/plone/recipe/zope2instance/ctl.py

```diff
@@ -27,7 +27,10 @@
 available actions.
 """
 
+from asyncore import dispatcher
```

Comment:
It seems that `asyncore` will be removed from Python's StdLib. That's why waitress 1.2.0b1 vendored it, maybe we should use already use the vendor version here.

See https://github.com/Pylons/waitress/blob/master/CHANGES.txt#L108-L110 for datails.

Reply:
good point, changed in https://github.com/plone/plone.recipe.zope2instance/pull/75/commits/02909cd371fe828ef3852f35408a30398f7e3ec4